docs/docsource/operation/Zenko_CLI/index.rst

```diff
@@ -31,4 +32,4 @@ or both.
    CRR_Pause_&_Resume
    Object_Lifecycle_Management
    Monitoring_NFS-SOFS_Ingestion
-
+   managing_bucket_policies
```

Comment:
Should this be capitalized?

Reply:
capitalization on file names is not consistent.